src/components/InfoSplitSection.svelte

```diff
@@ -20,21 +20,15 @@
     margin: 3rem;
   }
   section :global(p) {
-    font-size: 26px;
+    font-size: var(--body-xl);
```

Comment:
the current body-xl is 24 font. Are we just making standardizations right now regardless of previous fonts?

Reply:
Yep. If something's close enough, I'm just changing it

Reply:
Ye-ji actually asked for a consolidated list of values to update the mockups. Gave one to her but didn't fully implement it until now haha